parser/parser.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
+	"path/filepath"
 	"regexp"
 
 	"github.com/dogtools/dog/types"
 	"github.com/ghodss/yaml"
 )
 
 var ErrMalformedStringArray = errors.New("Malformed strings array")
+var ErrNoDogfiles = errors.New("No dogfiles found")
```

Comment:
Agreed! For what it's worth, the string in this error isn't actually displayed to the user at any point.

I agree that it's very confusing at the moment, and the code also makes no effort to differentiate both, as you can see by the `LoadDogFile` function which actually loads several Dogfiles and parses them as a Dogfile. This is even confusing to write about.